sormas-backend/src/main/java/de/symeda/sormas/backend/feature/FeatureConfiguration.java

```diff
@@ -101,4 +108,14 @@ public boolean isEnabled() {
 	public void setEnabled(boolean enabled) {
 		this.enabled = enabled;
 	}
+
+	@Type(type = ModelConstants.HIBERNATE_TYPE_JSON)
+	@Column(columnDefinition = ModelConstants.COLUMN_DEFINITION_JSON)
+	public Map<FeatureTypeProperty, Object> getProperties() {
```

Comment:
How are Maps working in the auditlog?

Reply:
I don't know, but the only way to change these values is to directly do it in the database anyway. If I'm not totally mistaken, there won't ever be audit log entries created as long as we don't provide a user interface for it.

Reply:
I agree. If we don't check this now, we will likely forget about it later.

Reply:
There's currently no way to check it specifically for this use case, but I can confirm that the epidemiological evidence criteria of events, which is a map annotated with the same annotations, is logged as well.